### PR TITLE
added my card information

### PR DESCRIPTION
**Pull request by nura-rikuo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,7 +114,35 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-	 
+
```
Comment (dilshankarunarathne) [inline index.html:117]: Looks like you did not include the comments at the start and end of the template.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
The code template was covered with 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->` and
`<!-- ________*END TEMPLATE* Contributor card END ________  -->` comments.
These comments are very helpful for future contributors and maintainers to read and understand the code. 
So, please include those comments in your card as well. 
And make sure to change `*TEMPLATE: MAKE A COPY*` and `*END TEMPLATE*` parts into your name.

**Comment by dilshankarunarathne** [comment]
Hello @nura-rikuo, 
we will be able to review and merge your pull request once you make these changes.

**Comment by Preet-Sojitra** [comment]
Congratulations @nura-rikuo for your contribution in open source. Best of luck for your future journey in open source :)

**Pull request merged**